src/cmds/docs/single.js

```diff
@@ -62,7 +62,7 @@ module.exports = class SingleDocCommand {
 
     debug(`selectedVersion: ${selectedVersion}`);
 
-    const createdDoc = await pushDoc(key, selectedVersion, dryRun, filepath);
+    const createdDoc = await pushDoc(key, selectedVersion, dryRun, filePath, this.category);
```

Comment:
Where's `this.category` being set?

Reply:
We pass that into the `pushDoc` function so we can use it for dynamic routing:

https://github.com/readmeio/rdme/blob/aaad56d984da47ed3576a8b672de5b848a07ff8d/src/lib/pushDoc.js#L42

The idea is we can use a single handler for `docs`, `changelogs`, and `custompages` routes

Reply:
OH right, `this.category` is the category that it shows up in the help screen.

Reply:
lol misread your question so my response didn't actually answer your question but yes, it's for the categories in the help screen